### PR TITLE
bundler: 2.3.20 -> 2.3.21

### PR DESCRIPTION
**Pull request by anthonyroussel**
###### Description of changes

* Upgrade bundler from 2.3.20 to 2.3.21
* Add passthru.updateScript to automatically update bundler
* Add missing meta information
* Add anthonyroussel to maintainers
  * There is no maintainers on this package
  * I use bundler really often, so I propose to add myself as a maintainer of the package

See release notes: https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2321-august-24-2022

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/ruby-modules/bundler/default.nix [hunk 1]
```diff
@@ -1,14 +1,31 @@
...
+  '';
+
   postFixup = ''
     sed -i -e "s/activate_bin_path/bin_path/g" $out/bin/bundle
   '';
+
+  meta = with lib; {
+    description = "Manage your Ruby application's gem dependencies";
+    homepage = "https://bundler.io";
+    license = "MIT";
```
Comment (marsam) [inline pkgs/development/ruby-modules/bundler/default.nix:28]:
```suggestion
    license = licenses.mit;
```
besides that, LGTM.

Comment (anthonyroussel) [reply pkgs/development/ruby-modules/bundler/default.nix:28]: Oops! Thanks for the review :)

Comment (anthonyroussel) [reply pkgs/development/ruby-modules/bundler/default.nix]: Fixed

**Review by marsam — APPROVED** [review]
LGTM. Thank you!

**Comment by vcunat** [comment]
As labeled by OfBorg, this was a huge rebuild, so I reverted it and moved to staging branch.

**Pull request merged**